src/RuntimePluginLoader.cpp

```diff
@@ -94,13 +94,13 @@ void RuntimePluginLoader::initPlugin(const std::string& path)
     printf("[RuntimePluginLoader]: dlopen failed: %s\n", dlerror());
   }
 
-  RuntimePluginLoader::Parent* (*getPlugin)() =
-      (RuntimePluginLoader::Parent * (*)()) dlsym(plugin, "getPlugin");
+  RuntimePluginLoader::Parent* (*RAJAGetPlugin)() =
+      (RuntimePluginLoader::Parent * (*)()) dlsym(plugin, "RAJAGetPlugin");
```

Comment:
This seems like an ABI breaking change. Do we plan to have any patch releases before the next major release? @davidbeckingsale @rhornung67

Reply:
suggestion was from @daboehme

Reply:
Maybe we can deprecate "getPlugin" but still look for it. Then remove it int eh next release.

Reply:
I have added the extra check for the old name just in case for now.

Reply:
@MrBurmark , I tried building with the latest change and I hit the following build error: 
```
RAJA/src/RuntimePluginLoader.cpp(120): error #20: identifier "error" is undefined
```

Reply:
That's what I get for making changes in the web interface.

Reply:
I assumed that we have a CI test that would find that, is that not the case?